src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/task/TaskCopyBuilder.java

```diff
@@ -86,7 +89,7 @@ public int execute(
               ContainerManager containerManager,
               Database database) {
             return copyFiles(sha1, taskId, copyId, taskConfig, taskDefLocation)
-                ? Job.STATUS_OK
+                ? STATUS_OK
```

Comment:
does this compile? STATUS_OK is not static imported so I'm confused about how this symbol is getting resolved. Also we should be consistent and either static import all the constants from Job or none of them